defactor-com/sdk
# Feat/contract publics

## Add new staking contract functions

### What does this PR do?

- Resolve #176 
- Resolve #177 
- Resolve #178 
- Resolve #179
- Resolve #180
- Resolve #181
- Resolve #182
- Resolve #183
- Resolve #184
- Resolve #185
- Resolve #186
- Resolve #187
- Resolve #188

### Steps to test

1. Enable running tests
2. Check all tests passes

#### CheckList
